source/cloud/gcp/gke.md

```diff
@@ -23,9 +23,7 @@ $ gcloud init
 Now we can launch a GPU enabled GKE cluster.
 
 ```console
-$ gcloud container clusters create rapids-gpu-kubeflow \
-  --accelerator type=nvidia-tesla-a100,count=2 --machine-type a2-highgpu-2g \
-  --zone us-central1-c --release-channel stable
+$ $ gcloud container clusters create rapids-gpu-kubeflow --accelerator type=nvidia-tesla-a100,count=2 --machine-type a2-highgpu-2g --zone us-central1-c --release-channel stable
```

Comment:
Yay I tested it on a local server and it works! I'll push those changes right now

Reply:
Ah great, glad that worked! I just tried on the docs preview and also confirmed it works as expected. And I think it's great that it now renders with the entire command visible:

<img width="757" alt="image" src="https://github.com/user-attachments/assets/9194f66a-d99c-4366-8f65-5f69e2c4db90">

Could you please open an issue documenting the work to remove these leading `$` from all the shell blocks across this repo?

I've always found them a little annoying anyway (because you have to manually exclude them when copying in a place that doesn't have the copy-button logic that filters them out), and now we have a specific functional reason to prefer not having them.

Don't actually do that implementation work though (except for other multi-line cases like this one where it's necessary to fix them), until @jacobtomlinson gives us his opinion on the issue.

Reply:
I have a few thoughts on this, @melodywang060 if you could open the issues that James suggested then we can discuss it over there.

Reply:
sounds good, I'll open it up right now 👍🏼

Reply:
New issue and documentation is here: https://github.com/rapidsai/deployment/issues/452